whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -69,7 +69,9 @@ public interface AttributeSyntax extends Documented {
             "\n" +
             "An as-set name can also be hierarchical.  A hierarchical set\n" +
             "name is a sequence of set names and AS numbers separated by\n" +
-            "colons \":\".  At least one component of such a name must be\n" +
+            "colons \":\". The first element of the name must be an ASN\n" +
+            "followed by colons and finishing with as-set (example: AS15562:AS-SNIJDERS)." +
```

Comment:
We should always use generic examples (i.e. don't reference a specific person or company). How about AS3333:AS-TEST as an example instead?